spec/support/blueprints.rb

```diff
@@ -96,9 +113,23 @@
   total { 1_000_00 }
 end
 
+Purchase.blueprint(:invalid) do
```

Comment:
In fact what make this Purchase invalid is the fact that some offer related to this is expired, nothing more.